sbord/sbord.py

```diff
@@ -13,13 +13,15 @@
 import plotly.graph_objects as go
 from natsort import natsorted
 from scipy.signal import savgol_filter
+import subprocess
+import cv2
```

Comment:
Not needed, is it? Could you delete it to reduce dependencies?

Reply:
sure, just a remnant I forgot to delete :)

Reply:
should be removed by now